rapids-cmake/cpm/versions.json

```
@@ -33,9 +33,8 @@
     "rapids_logger": {
       "version": "0.1.0",
       "git_shallow": false,
-      "always_download": true,
       "git_url": "https://github.com/rapidsai/rapids-logger.git",
-      "git_tag": "8968ab3337f31c845d4e3bf6c55ae89242ded22b"
+      "git_tag": "main"
```

This suggestion should be accepted before merging. If we need any more commits in the rapids-logger repo we can bump the commit then.
```suggestion
      "git_tag": "d8a7ba8a105163eb2dde42a8e71a1cb3680100e2"
```